scripts/west_commands/genboard/ncs_genboard.py

```diff
@@ -119,7 +122,7 @@ def do_run(self, args, unknown_args):
                 break
 
         if not targets:
-            log.err(f"Invalid/unsupported variant: {args.variant}")
+            log.err(f"Invalid/unsupported variant: {req_variant}")
```

Comment:
Does it make sense to validate `soc` and `variant` after the validation in line 62. passed? All supported combinations of socs and variants can be listed in the schema.

Reply:
this means the internal config file is not aligned, should really be catched in tests...